core/drivers/crypto/hisilicon/hpre_rsa.h

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2024, HiSilicon Technologies Co., Ltd.
+ */
+
+#ifndef __TEE_HPRE_RSA_H__
+#define __TEE_HPRE_RSA_H__
+
+#include <stdint.h>
+#include <tee_api_types.h>
+#include <types_ext.h>
+
+#define PKCS_V1_5_MSG_MIN_LEN			11
+#define PKCS_V1_5_PS_MIN_LEN			8
+#define PKCS_V1_5_PS_POS			2
+#define PKCS_V1_5_FIXED_LEN			3
+#define PKCS_V1_5_PS_DATA			0x5a
+#define OAEP_MAX_HASH_LEN			64
+#define OAEP_MAX_DB_LEN				512
+#define PRIME_QUALITY_FLAG			1024
+#define HPRE_RSA_GEN_TOTAL_BUF_SIZE(kbytes)	((kbytes) * 7)
```

Comment:
This macro is unused. Will it be needed in later patches?

Reply:
solved